docs/Using-Data-Access-Tokens.md

```diff
@@ -0,0 +1,10 @@
+#### What are Data Access Tokens?
```

Comment:
How does the markdown style integrate with the existing Web API page?  Are they consistent?

Reply:
Not exactly. So I've changed things around so that the content now originates in the same way (it is part of the webAPI static page content).